src/CryptoHelper.php

```diff
@@ -8,6 +8,15 @@
  */
 class CryptoHelper
 {
+    protected static function stringIsBinary($str): bool
+    {
+        $str = str_ireplace("\t", "", $str);
+        $str = str_ireplace("\n", "", $str);
+        $str = str_ireplace("\r", "", $str);
```

Comment:
You could just do `str_ireplace(["\t", "\n", "\r"], "", $str)`

Reply:
👍